src/v/archival/upload_housekeeping_service.cc

```diff
@@ -134,8 +140,7 @@ ss::future<> upload_housekeeping_service::bg_idle_loop() {
 
 void upload_housekeeping_service::rearm_idle_timer() {
     _idle_timer.rearm(
-      ss::lowres_clock::now() + _idle_timeout()
-      + _time_jitter.next_jitter_duration());
+      ss::lowres_clock::now() + _idle_jittery_timeout.next_duration());
```

Comment:
@Lazin would you mind reviewing this commit, to make sure the idle timer handling is consistent with what you had in mind when writing the class?